src/Domain/Common/AuditEntity.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
```

Comment:
Something seems to be incorrectly configured here. I get the following warnings when building the solution:

~~~none
Entities/TagHistory.cs(9,21): warning CS0108: 'TagHistory.HistoryId' hides inherited member 'AuditEntity<Tag>.HistoryId'. Use the new keyword if hiding was intended. [/home/me/dev/codidact/core/src/Domain/Domain.csproj]
Entities/TagHistory.cs(10,25): warning CS0108: 'TagHistory.HistoryActivityAt' hides inherited member 'AuditEntity<Tag>.HistoryActivityAt'. Use the new keyword if hiding was intended. [/home/me/dev/codidact/core/src/Domain/Domain.csproj]
Entities/TagHistory.cs(13,25): warning CS0108: 'TagHistory.CreatedAt' hides inherited member 'AuditableEntity.CreatedAt'. Use the new keyword if hiding was intended. [/home/me/dev/codidact/core/src/Domain/Domain.csproj]
Entities/TagHistory.cs(14,25): warning CS0108: 'TagHistory.LastModifiedAt' hides inherited member 'AuditableEntity.LastModifiedAt'. Use the new keyword if hiding was intended. [/home/me/dev/codidact/core/src/Domain/Domain.csproj]
Entities/TagHistory.cs(15,21): warning CS0108: 'TagHistory.CreatedByMemberId' hides inherited member 'AuditableEntity.CreatedByMemberId'. Use the new keyword if hiding was intended. [/home/me/dev/codidact/core/src/Domain/Domain.csproj]
Entities/TagHistory.cs(16,21): warning CS0108: 'TagHistory.LastModifiedByMemberId' hides inherited member 'AuditableEntity.LastModifiedByMemberId'. Use the new keyword if hiding was intended. [/home/me/dev/codidact/core/src/Domain/Domain.csproj]
Entities/CategoryHistory.cs(16,21): warning CS0108: 'CategoryHistory.HistoryId' hides inherited member 'AuditEntity<Category>.HistoryId'. Use the new keyword if hiding was intended. [/home/me/dev/codidact/core/src/Domain/Domain.csproj]
Entities/CategoryHistory.cs(17,25): warning CS0108: 'CategoryHistory.HistoryActivityAt' hides inherited member 'AuditEntity<Category>.HistoryActivityAt'. Use the new keyword if hiding was intended. [/home/me/dev/codidact/core/src/Domain/Domain.csproj]
Entities/CategoryHistory.cs(18,21): warning CS0108: 'CategoryHistory.HistoryActivityMemberId' hides inherited member 'AuditEntity<Category>.HistoryActivityMemberId'. Use the new keyword if hiding was intended. [/home/me/dev/codidact/core/src/Domain/Domain.csproj]
Entities/CategoryHistory.cs(19,31): warning CS0114: 'CategoryHistory.HistoryActivityMember' hides inherited member 'AuditEntity<Category>.HistoryActivityMember'. To make the current member override that implementation, add the override keyword. Otherwise add the new keyword. [/home/me/dev/codidact/core/src/Domain/Domain.csproj]
~~~

I do not fully understand what is going on here, but it seems something was not generated properly here.

Reply:
Well since it isn't generated obviously I just missed this.
There was also a typo in TagHistory with the column being `HistoryActivityId` and not `HistoryActivityMemberId`.
Many such things were found after editing the files.